saleor/graphql/checkout/mutations/checkout_shipping_method_update.py

```diff
@@ -54,6 +55,16 @@ class Meta:
         doc_category = DOC_CATEGORY_CHECKOUT
         error_type_class = CheckoutError
         error_type_field = "checkout_errors"
+        webhook_events_info = [
+            WebhookEventInfo(
+                type=WebhookEventSyncType.SHIPPING_LIST_METHODS_FOR_CHECKOUT,
+                description="Triggered when provided delivery method is external id.",
```

Comment:
Same as earlier: "Triggered when provided delivery method is an external one, provided by this webhook."